central/compliance/service/service_impl.go

```diff
@@ -144,6 +151,11 @@ func (s *serviceImpl) GetRunResults(ctx context.Context, request *v1.GetComplian
 	if err != nil {
 		return nil, err
 	}
+	config, exsist, _ := s.complianceDataStore.GetConfig(ctx, request.GetStandardId())
```

Comment:
If a user is asking for specific results, then we should not deny them the results as they are specifically asking for them

Reply:
@connorgorman @ksurabhi91 The current contract of `hideScanResults` leaves room for different interpretations. In that, whether the admin is using it to actively hide results from the consumers or is meant to exclude scan results from only certain APIs. Thus, if the interpretation is former, then the implementation is in the PR; Otherwise, Connor's point is valid. This can lead to confusion. For instance, users may complain about why they cannot view the results on some endpoints, while they can fetch it directly if we follow Connor's argument.

I think we should look at options to resolve the confusion.

Reply:
@md2119  I think it is the former. Use case is that user does not want to see results from standards which are irrelevant.